tests/test_rag.py

```diff
@@ -1,13 +1,17 @@
+import logging
 import os
 from collections import namedtuple
 
 import pytest
 
+from clarifai.client import User
 from clarifai.rag import RAG
 from clarifai.urls.helper import ClarifaiUrlHelper
 
 CREATE_APP_USER_ID = os.environ["CLARIFAI_USER_ID"]
 
+TEXT_FILE_PATH = os.path.dirname(__file__) + "/assets/sample.txt"
```

Comment:
Seems like this file is missing from the PR.

Reply:
Its already in the tests asset folder